dqc_us_rules/dqc_us_0033_0036.py

```diff
@@ -39,57 +56,124 @@ def _doc_period_end_date_check(modelXbrl):
             continue
         fact_eop_date = dateunionDate(eop_fact.xValue, subtractOneDay=True)
         # Arelle adjusts context end date to end-of-day midnight
-        # Reverse the adjustment to get the expected date value by subtracting one day
-        context_eop_date = dateunionDate(eop_context.endDatetime, subtractOneDay=True)
+        # Reverse the adjustment to get the expected date value
+        # by subtracting one day
+        context_eop_date = dateunionDate(
+            eop_context.endDatetime,
+            subtractOneDay=True
+        )
         delta = context_eop_date - fact_eop_date
         if abs(delta.days) > 3:
             is_valid_dped = False
-            result_group.append(('{}.1'.format(_CODE_NAME_36), messages.get_message(_CODE_NAME_36), context_eop_date, eop_fact, default_dped_fact))
-
-    if is_valid_dped:  # Don't loop through them if the DPED date is bad, since the date is incorrect.
-        # loop through the dei facts and compare against their LEA's DocumentPeriodEndDate
+            result_group.append((
+                '{}.1'.format(_CODE_NAME_36),
+                messages.get_message(_CODE_NAME_36),
+                context_eop_date,
+                eop_fact,
+                default_dped_fact
+            ))
+
+    # Don't loop through them if the DPED date is bad, since the date
+    # is incorrect.
+    if is_valid_dped:
+        # loop through the dei facts and compare against their LEA's
+        # DocumentPeriodEndDate
         for lea_key, fact_group in dei_facts.items():
             eop_fact = dped_facts.get(lea_key, default_dped_fact)[0]
-            if eop_fact is None or eop_fact.context is None or eop_fact.context.endDatetime is None:
+            if (eop_fact is None or
```

Comment:
SG suggestion to match our multiline if:
```
if ((eop_fact is None or
     eop_fact.context is None or
     eop_fact.context.endDatetime is None)):
    continue
```
This is to keep everything aligned correctly and break up the indention from the multi-line if and it's body.